rust/agama-cli/src/logs.rs

```diff
@@ -22,10 +23,25 @@ pub enum LogsCommands {
     List,
 }
 
+// main entry point called from agama CLI main loop
 pub async fn run(subcommand: LogsCommands) -> anyhow::Result<()> {
     match subcommand {
-        LogsCommands::Store { verbose } => Ok(store(verbose)?),
-        LogsCommands::List => Err(anyhow::anyhow!("Not implemented")),
+        LogsCommands::Store { verbose } => {
+            // feed internal options structure by what was received from user
+            // for now we always use / add defaults if any
+            let mut options = LogOptions::new();
+
+            options.verbose = verbose;
+
+            Ok(store(options)?)
+        }
+        LogsCommands::List => {
+            let options = LogOptions::new();
```

Comment:
You can change this to (if Default is implemented):

```rust
            list(LogOptions::default());
            Ok(())
```

Reply:
done